### ISSUE TITLE
[feature request]: Copy Rufus Windows installation settings

### ISSUE THREAD

**Issue by davidhedlund** [ventoy/Ventoy#2637]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.96

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

Yes. I have tried them, but the bug still exist.

### BIOS Mode

Legacy BIOS Mode

### Partition Style

MBR

### Disk Capacity

32

### Image file checksum (if applicable)

None

### What happened?

![image](https://github.com/ventoy/Ventoy/assets/6416073/64d08868-937a-43d7-905f-195b23c905e6)

![Screenshot 2023-10-27 121427](https://github.com/ventoy/Ventoy/assets/6416073/882b0eac-01fc-4702-a500-580d2db29619)

**Comment by davidhedlund** [comment]
* https://github.com/ventoy/Ventoy/issues/2639
* https://github.com/ventoy/Ventoy/issues/2638

**Comment by catherinedoyel** [comment]
What you could do if you always wanted the following changes is use Rufus to make a Windows installer the way you like it. Then make a copy of the original .iso file and open it with 7zip or similar software. Then copy from the files from the rufus installation to the iso overwriting differences.

**Comment by davidhedlund** [comment]
> What you could do if you always wanted the following changes is use Rufus to make a Windows installer the way you like it. Then make a copy of the original .iso file and open it with 7zip or similar software. Then copy from the files from the rufus installation to the iso overwriting differences.

That would work somehow. But ideally, you make a disk image of a small USB flash drive that fulfills the hardware requirement for Windows, install it with Rufus, and then make a disk image. However, Rufus is Windows-specific, while Ventoy is cross-platform.

**Comment by krystofkrticka** [comment]
Yes those settings would be amazing if they were in ventoy especially "Disable data collection" and "Disable bitlocker automatic device encryption". Especially the disable bitlocker automatic device encryption would solve so many issues because it would prevent windows from automatically encrypting users drive without their knowledge, this is probably familiar if you read news that microsoft is thinking about encrypting all drives for users by default.
